src/Terminal/HelpText.ts

```diff
@@ -336,14 +336,16 @@ export const HelpTexts: Record<string, string[]> = {
   ],
   killall: ["Usage: killall", " ", "Kills all scripts on the current server."],
   ls: [
-    "Usage: ls [dir] [-l] [--grep pattern]",
+    "Usage: ls [dir] [-l] [-h] [-g, --grep pattern]",
     " ",
     "The ls command, with no arguments, prints all files and directories on the current server's directory to the Terminal screen. ",
     "The files will be displayed in alphabetical order. ",
     " ",
     "The 'dir' optional parameter can be used to display files/directories in another directory.",
     " ",
-    "The '-l' optional parameter allows you to force each item onto a single line.",
+    "The '-l' optional parameter allows you to force each item onto a single line, displaying two new columns for the script's RAM usage and filesize.",
```

Comment:
```suggestion
    "The '-l' optional parameter allows you to force each item onto a single line, displaying two columns for the script's RAM usage and filesize.",
```
Help text/comments should be about the current state, not the history.